src/PasswordHasher/LegacyPasswordHasher.php

```diff
@@ -29,11 +29,14 @@ class LegacyPasswordHasher extends AbstractPasswordHasher
 {
     /**
      * Default config for this object.
+     * - `hashType` String identifier of the hash type to use on the password. (e.g 'sha256' or 'md5')
+     * - `salt` Boolean flag for salting the password in a hashes, and checks.
```

Comment:
If this is supposed to be boolean, why not set the default value to `true` instead of `null`?

Reply:
Good point, I changed the default to true.